h/models/organization.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+
+
+import sqlalchemy as sa
+import slugify
+
+from h.db import Base
+from h.db import mixins
+
+
+ORGANIZATION_NAME_MIN_LENGTH = 4
+ORGANIZATION_NAME_MAX_LENGTH = 25
+ORGANIZATION_LOGO_MAX_LENGTH = 3000
```

Comment:
Hi! Presuming that a logo consists of inline SVG markup. How did you decide on a 3000-character max? Also, perhaps a comment here to indicate what `3000` means (i.e. that it's a character limit for SVG markup)?

Reply:
Yeah I can imagine once we have self-serve uploading we may need some sort of a size limit. For now though I think I'd probably recommend leaving it without a limit